pyproject.toml

```diff
@@ -27,6 +27,8 @@ dependencies = [
     "spurplus ~= 2.3.4",
     "websockets ~= 10.3",
     "charset_normalizer ~= 2.1.1",
+    "requests",
```

Comment:
Move the tool into Azure folder, and the dependencies should be in Azure too.

Reply:
Create a new installation part called "ado" like azure, aws. And check if it's possible to add it as a depenency of azure.